### ISSUE TITLE
Using #alias_attribute with :id breaks #update_attribute 

### ISSUE THREAD

**Issue by jamesBrennan** [composite-primary-keys/composite_primary_keys#450]
Using the method `alias_attribute` to alias a column to `:id` breaks `update_attribute`.

Here is a [failing spec](https://github.com/jamesBrennan/composite_primary_keys/commit/93a5d731b91e29ffb5d23d0e17732264f1584194) to illustrate the bug.

```ruby
# test/test_alias_attribute.rb
require File.expand_path('../abstract_unit', __FILE__)

class TestAliasAttribute < ActiveSupport::TestCase
  fixtures :departments

  def test_alias_attribute_to_id
    Department.class_eval do
      alias_attribute :id, :department_id
    end

    error = nil
    Department.find([1,1]).update_attribute(:location_id, 3)
  rescue StandardError => e
    error = e
  ensure
    assert_nil(error)
  end
end
```

`rake postgresql:test TEST=test/test_alias_attribute.rb` fails with this error:

    Expected #<NoMethodError: undefined method `each_with_index' for 1:Integer> to be nil.

**Comment by cfis** [comment]
Guess I'm not particularly surprised.  Is there some reason you would want to do this though?

**Comment by cfis** [comment]
Added the example test, which now works as it should. Hopefully this was fixed in b0413f5 or earlier commits.
Thanks for the report.
